storages/backends/ftp.py

```diff
@@ -236,7 +252,6 @@ def read(self, num_bytes=None):
         if not self._is_read:
             self._storage._start_connection()
             self.file = self._storage._read(self._name)
-            self._storage._end_connection()
```

Comment:
function _end_connection() doesn't exist. With current code read() doesn't work at all because of this.